get_aqi.py

```diff
@@ -1,32 +1,55 @@
 
 def parse_args():
     parser = argparse.ArgumentParser(description="Measure air quality using an SDS011 sensor.")
 
-    parser.add_argument("--country", "-c", choices=["CN", "EU", "US"], default="EU", metavar="COUNTRY", help="country code (ISO 3166-1 alpha-2) used to compute AQI. Currently accepted values (default: EU) : 'CN' (AQI Mainland China), 'EU' (CAQI) and 'US' (AQI US)")
-    parser.add_argument("--delay", "-d", default=1200, metavar="SECONDS", type=int, help="seconds to pause after getting data with the sensor before taking another measure (default: 1200, ie. 20 minutes)")
+    parser.add_argument("-b", "--bind", metavar='ADDRESS', default='0.0.0.0', help="Specify alternate bind address [default: 0.0.0.0]")
+    parser.add_argument("-p", "--port", metavar='PORT', default=8000, type=int, help="Specify alternate port [default: 8000]")
+    parser.add_argument("--country", "-c", choices=["CN", "EU", "US"], default="US", metavar="COUNTRY", help="country code (ISO 3166-1 alpha-2) used to compute AQI. Currently accepted values (default: EU) : 'CN' (AQI Mainland China), 'EU' (CAQI) and 'US' (AQI US)")
+    parser.add_argument("--delay", "-d", default=10, metavar="SECONDS", type=int, help="seconds to pause after getting data with the sensor before taking another measure (default: 1200, ie. 20 minutes)")
     parser.add_argument("--log", "-l", metavar="FILE", help="path to the CSV file where data will be appended")
     parser.add_argument("--measures", "-m", default=3, metavar="N", type=int, help="get PM2.5 and PM10 values by taking N consecutive measures (default: 3)")
     parser.add_argument("--mqtt-hostname", "-n", metavar="IP/HOSTNAME", help="IP address or hostname of the MQTT broker")
     parser.add_argument("--mqtt-port", "-r", default="1883", metavar="PORT", type=int, help="Port number of the MQTT broker (default: '1883')")
-    parser.add_argument("--mqtt-base-topic", "-i", default="aqi", metavar="TOPIC", help="Parent MQTT topic to use (default: 'aqi')")
+    parser.add_argument("--mqtt-base-topic", "-i", default="sds011", metavar="TOPIC", help="Parent MQTT topic to use (default: 'aqi')")
     parser.add_argument("--omnia-leds", "-o", action="store_true", help="set Turris Omnia LED colors according to measures (User #1 LED for PM2.5 and User #2 LED for PM10)")
     parser.add_argument("--sensor", "-s", default="/dev/ttyUSB0", metavar="FILE", help="path to the SDS011 sensor (default: '/dev/ttyUSB0')")
-    parser.add_argument("--sensor-operation-delay", "-p", default=1, metavar="SECONDS", type=int, help="seconds to let the sensor start (default: 10)")
-    parser.add_argument("--sensor-start-delay", "-t", default=10, metavar="SECONDS", type=int, help="seconds to let the sensor perform an operation : taking a measure or going to sleep (default: 1)")
+    parser.add_argument("--sensor-operation-delay", "-e", default=1, metavar="SECONDS", type=int, help="seconds to let the sensor sleep (default: 1)")
```

Comment:
Sorry, but the defaults values and help messages of  the `--sensor-operation-delay` and `--sensor-start-delay` are inverted in the current release. Should be :

```suggestion
    parser.add_argument("--sensor-operation-delay", "-e", default=1, metavar="SECONDS", type=int, help="number of seconds to let the sensor perform an operation : taking a measure or going to sleep (default: 1)")
```